src/main/scala/bloop/integrations/gradle/BloopPlugin.scala

```diff
@@ -36,4 +82,72 @@ final class BloopPlugin extends Plugin[Project] {
     bloopInstall.dependsOn(configureBloopInstall)
     ()
   }
+
+  private def findCompatibleConfigNamesFromSourceSet(sourceSet: SourceSet): Set[String] = Set(
+    sourceSet.getApiConfigurationName(),
+    sourceSet.getImplementationConfigurationName(),
+    sourceSet.getCompileOnlyConfigurationName(),
+    // sourceSet.getCompileOnlyApiConfigurationName(),
+    sourceSet.getCompileClasspathConfigurationName(),
+    sourceSet.getRuntimeOnlyConfigurationName(),
+    sourceSet.getRuntimeClasspathConfigurationName(),
+    sourceSet.getRuntimeElementsConfigurationName(),
+    "scalaCompilerPlugins"
```

Comment:
I think only `getCompileClasspathConfigurationName()`, `getRuntimeClasspathConfigurationName()` and `scalaCompilerPlugins` is needed here as the others aren't resolvable. See [tables](https://docs.gradle.org/current/userguide/java_library_plugin.html)